tock/projects/tests.py

```diff
@@ -494,13 +494,11 @@ def test_total_hours_billed(self):
         TimecardObject.objects.create(
             timecard = Timecard.objects.get(pk=1),
             project=Project.objects.get(pk=1),
-            submitted = True,
             hours_spent= 10
         )
         TimecardObject.objects.create(
             timecard = Timecard.objects.get(pk=2),
             project=Project.objects.get(pk=1),
-            submitted = False,
```

Comment:
I don't understand why these `submitted` values were set in the first place, as far as I can tell, this doesn't impact the test in any way. (To be clear, not sure why one is True, and the other is False)

Reply:
Me either, I suspect a well intentioned but abandoned functionality path